src/Uplink/Admin/Notice.php

```diff
@@ -16,12 +16,12 @@ class Notice {
 	/**
 	 * @var array<mixed>
 	 */
-	protected array $saved_notices = [];
```

Comment:
These were added previously by another developer, technically this project does not support the minimum of [PHP7.4](https://github.com/stellarwp/uplink/blob/29306946e41fcc21d3c8a44801e62789acc55705/composer.json#L31)